config/database.yml

```diff
@@ -4,8 +4,8 @@ development: &dev
   database: ls_development
   host: <%= ENV['DB_HOST'] || ""%>
```

Comment:
what do you think about add "localhost" as default to host?

Reply:
yep makes sense, not sure why it wasnt set so 🤔

Reply:
Done